brave-spancollector-kafka/src/main/java/com/github/kristofa/brave/kafka/KafkaSpanCollector.java

```diff
@@ -0,0 +1,100 @@
+package com.github.kristofa.brave.kafka;
+
+import com.github.kristofa.brave.SpanCollector;
+
+import java.io.Closeable;
+import java.util.Properties;
+import java.util.concurrent.*;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import com.twitter.zipkin.gen.Span;
+import kafka.javaapi.producer.Producer;
+import kafka.producer.ProducerConfig;
+
+/**
+ * SpanCollector which submits spans to Kafka using <a href="http://kafka.apache.org/documentation.html#producerapi">Kafka Producer api</a>.
+ * <p>
```

Comment:
I'm not actually sure this is correct. As written it looks like the Keyed message is given the topic of "zipkin" and uses a null key.  Eventually it may make sense to be able to configure a topic other than zipkin, as opposed to hard coding it.

You wouldn't want to set the key here to "zipkin" as well since that would mean all messages would hash to the same kafka partition.  Using a null key is better as long as order isn't important, because null will hash randomly to each partition however order is only guaranteed per partition.